app/javascript/app/document-capture/components/acuant-capture.jsx

```diff
@@ -38,24 +62,41 @@ function AcuantCapture({ label, hint, bannerText, value, onChange, className })
         </FullScreen>
       )}
       <FileInput
+        ref={inputRef}
         label={label}
         hint={hint}
         bannerText={bannerText}
         accept={['image/*']}
         value={value}
-        onClick={startCaptureIfSupported}
+        onClick={startCaptureOrTriggerUpload}
         onChange={onChange}
       />
-      {hasCapture && (
-        <Button
-          isSecondary={!value}
-          isUnstyled={!!value}
-          onClick={() => setIsCapturing(true)}
-          className="display-block margin-top-2"
-        >
-          {t(value ? 'doc_auth.buttons.take_picture_retry' : 'doc_auth.buttons.take_picture')}
-        </Button>
-      )}
+      <div className="margin-top-2">
+        {isMobile && (
+          <Button
+            isSecondary={!value}
+            isUnstyled={!!value}
+            onClick={startCaptureOrTriggerUpload}
+            className={`margin-right-${value ? 1 : 2}`}
+          >
+            {hasCapture &&
+              (value
+                ? t('doc_auth.buttons.take_picture_retry')
+                : t('doc_auth.buttons.take_picture'))}
+            {!hasCapture && t('doc_auth.buttons.upload_picture')}
+          </Button>
+        )}
+        {isMobile &&
+          hasCapture &&
+          formatHTML(t('doc_auth.buttons.take_or_upload_picture'), {
+            'lg-take-photo': () => null,
```

Comment:
making this null essentially removes it the "take photo" line right? should we just break these into separate translations instead?

Reply:
> making this null essentially removes it the "take photo" line right? should we just break these into separate translations instead?

I'd meant to add some clarifying comment here since, yes, the `lg-take-photo` element from the original string is essentially discarded.

In case it's what you'd meant by separate translations, my understanding of localization is that trying to chunk up strings into fragments like "or upload photo" can be problematic, since from the perspective of a translator, the context is not obvious, and the translation could potentially change (ordering, etc) depending on how best the fragments be combined in the language.

An alternative could be to format the button above (the "Take Picture" button) into `lg-take-photo`. I thought it might make the logic a little cumbersome, since this string is only used in specific circumstances (mobile, has capture) which aren't the same as when the "Take Picture" button should be present (mobile, whether or not capture is supported). That being said, one advantage of assigning the string this way is that we could choose to do that in the future as a refactoring, since that's essentially what the `lg-take-photo` element is meant to be represented by.

Reply:
that seems like a good enough reason, especially since we don't have another way to attach context to strings for translators, let's keep it as-is